### PR TITLE
Fpo 427 learning rate warm up

### PR DESCRIPTION
**Pull request by gurdipk**
### What?

I have added/removed/altered:
- learning rate warm up
- amended dropout 1 proportion to 0.1

### Why?

I am doing this because:
- Better predictability results seen from above changes on benchmarks

### CODE REVIEW

File: training/train_model.py [hunk 1]
```diff
@@ -46,6 +46,16 @@ def run(
         logger.info("Created model")
         logger.info(model)
 
+        ###Learning rate warm up
+        def lr_lambda(current_step: int):
+            warmup_steps = 0.05 * total_steps  # 5% of total steps for warmup
```
Comment (willfish) [inline training/train_model.py:51]: Probably we can make the warmup percentage configurable

**Review by willfish — APPROVED** [review]
